starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -799,4 +800,24 @@ protected boolean getIsAmatiDevice() {
   protected String getBuildFingerprint() {
     return Build.FINGERPRINT;
   }
+
+  @SuppressWarnings("unused")
+  @UsedByNative
+  protected long getPlayServicesVersion() {
+    try {
+      if (android.os.Build.VERSION.SDK_INT < 28) {
+        return appContext
+            .getPackageManager()
+            .getPackageInfo(GOOGLE_PLAY_SERVICES_PACKAGE, 0)
```

Comment:
I heard from android people, they say getPackageInfo can be slow. If this will be called multiple times, we probably should cache the result, also check if this could impact the start up latency.

Reply:
It's not synchronously being called though i hope ? Wouldn't want to delay app startup on this